File: app/src/main/java/br/com/onibusapp/onibusapp/adapter/RecyclerViewOnClickListener.java
package br.com.onibusapp.onibusapp.adapter;

import android.view.View;

/**
 * Created by diego on 24/07/2018.
 */

public interface RecyclerViewOnClickListener {

    void onClickListener(View view, int position);

    void onLongPressClickListener(View view, int position);

}

File: app/src/main/java/br/com/onibusapp/onibusapp/adapter/RecyclerViewOnClickListener.kt
package br.com.onibusapp.onibusapp.adapter

import android.view.View

/**
 * Created by diego on 24/07/2018.
 */

interface RecyclerViewOnClickListener {

    fun onClickListener(view: View, position: Int)

    fun onLongPressClickListener(view: View, position: Int)

}
